test/base/populators.py

```diff
@@ -1187,7 +1188,7 @@ class GiPostGetMixin(object):
     """Mixin for adapting Galaxy testing populators helpers to bioblend."""
 
     def _get(self, route, data={}):
```

Comment:
Do you want also drop the mutable default ?

Reply:
I should indeed, I'll merge this and open a follow up.